tensorflow-xla-runtime.spec

```diff
@@ -12,8 +12,13 @@ cp -r ${PY3_TENSORFLOW_ROOT}/lib/python%{cms_python3_major_minor_version}/site-p
 
 export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
 
+%define cxxflags -fPIC
```

Comment:
thanks @iarspider. Can you please include `## INCLUDE compilation_flags` and add `%{?arch_build_flags}` in the `cxxflags` here.